### DISCUSSION TITLE
datmodule.aug() ?

### DISCUSSION THREAD

**Discussion by Atmoboran** [terrastackai/terratorch#619] [Q&A]
hello,

what does '.aug' exactly do in the test plots?

So I was plotting my test data with my own routine and didn't include it at the beginning. This resulted in good (approx. what one would expect from the losses and accuracies) visual results for the models with pretrained_backbone = False, but very bad (much worse than the losses and accuracies would suggest) visual results for the models with pretrained_backbone = True. 

    with torch.no_grad():
        batch = next(iter(test_loader))
        **images = datamodule.aug(batch)**
        images = batch["image"].to(model.device)
        masks = batch["mask"].numpy()

Best regards,
Boran
